playbooks/robusta_playbooks/krr.py

```diff
@@ -77,11 +77,17 @@ def priority(self) -> int:
         return krr_severity_to_priority(self.severity)
 
 
+class KRRStrategyData(BaseModel):
+    name: str
+    settings: dict[str, Any]
+
+
 class KRRResponse(BaseModel):
     scans: List[KRRScan]
     score: int
     resources: List[ResourceType] = ["cpu", "memory"]
-    description: Optional[str] = None
+    description: Optional[str] = None # Old KRR versions do not return this field.
```

Comment:
I would just write which version of KRR this is added in or else all the new fields will have this comment